File: 0684/sesión15 - consulta/SoftVen/SoftVenModel/VendedoresDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftVen.Model
{
    public class VendedoresDTO
    {
        private int? vendedorId;
        private string nombre;
        private string paterno;
        private string materno;
        private TiendasDTO tienda;

        public VendedoresDTO()
        {
            this.VendedorId = null;
            this.Nombre = null;
            this.Paterno = null;
            this.Materno = null;
            this.Tienda = null;
        }

        public VendedoresDTO(int vendedorId, string nombre, string paterno, string materno, TiendasDTO tienda)
        {
            this.VendedorId = vendedorId;
            this.Nombre = nombre;
            this.Paterno = paterno;
            this.Materno = materno;
            this.Tienda = tienda;
        }

        public int? VendedorId { get => vendedorId; set => vendedorId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Paterno { get => paterno; set => paterno = value; }
        public string Materno { get => materno; set => materno = value; }
        public TiendasDTO Tienda { get => tienda; set => tienda = value; }
    }
}

File: 0684/sesión20 - WS - RESTFul/Back-End/ServidorRESTFul/SoftVenModel/src/main/java/pe/edu/pucp/softven/model/VendedoresDTO.java
package pe.edu.pucp.softven.model;

public class VendedoresDTO {

    private Integer vendedorId;
    private String nombre;
    private String paterno;
    private String materno;
    private TiendasDTO tienda;

    public VendedoresDTO() {
        this.vendedorId = null;
        this.nombre = null;
        this.paterno = null;
        this.materno = null;
        this.tienda = null;
    }

    public VendedoresDTO(Integer vendedorId, String nombre, String paterno, String materno, TiendasDTO tienda) {
        this.vendedorId = vendedorId;
        this.nombre = nombre;
        this.paterno = paterno;
        this.materno = materno;
        this.tienda = tienda;
    }

    public Integer getVendedorId() {
        return vendedorId;
    }

    public void setVendedorId(Integer vendedorId) {
        this.vendedorId = vendedorId;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPaterno() {
        return paterno;
    }

    public void setPaterno(String paterno) {
        this.paterno = paterno;
    }

    public String getMaterno() {
        return materno;
    }

    public void setMaterno(String materno) {
        this.materno = materno;
    }

    public TiendasDTO getTienda() {
        return tienda;
    }

    public void setTienda(TiendasDTO tienda) {
        this.tienda = tienda;
    }

}
